examples/targeting/add_customer_negative_criteria.rb

```diff
@@ -0,0 +1,111 @@
+#!/usr/bin/env ruby
+# Encoding: utf-8
+#
+# Copyright:: Copyright 2017, Google Inc. All Rights Reserved.
+#
+# License:: Licensed under the Apache License, Version 2.0 (the "License");
+#           you may not use this file except in compliance with the License.
+#           You may obtain a copy of the License at
+#
+#           http://www.apache.org/licenses/LICENSE-2.0
+#
+#           Unless required by applicable law or agreed to in writing, software
+#           distributed under the License is distributed on an "AS IS" BASIS,
+#           WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or
+#           implied.
+#           See the License for the specific language governing permissions and
+#           limitations under the License.
+#
+# This example adds various types of negative criteria to a customer. These
+# criteria will be applied to all campaigns for the customer.
```

Comment:
I think the second sentence here is a little confusing. It sounds like you'll be manually applying criteria to each campaign, rather than that they will automatically apply to each campaign since they're set at the customer level. Can we make sure it's clear what exactly is going to happen?

Reply:
this was copied verbatim from the soap example: https://github.com/googleads/google-api-ads-ruby/blob/master/adwords_api/examples/v201809/targeting/add_customer_negative_criteria.rb#L19. Do you have copy you'd prefer?